### DISCUSSION TITLE
Links.

### DISCUSSION THREAD

**Discussion by Ironbrodiet** [NebulaServices/Nebula#86] [General]
I know you said to not 'beg' for links in the issues so instead I came here.  Are there any links I can get that aren't replit, cylic or render related? (my school blocked those). And if you tell me to go to the discord, I'll just say I can't because I need a proxy for it, and that's exactly what I'm asking for rn :).

**Comment by cloverdata** [comment]
We have links in our discord, otherwise you can deploy it yourself it's not too hard.  https://docs.nebulaproxy.io/services/nebulaweb/deployment.html

**Reply by GoofyGoob123** [reply]
didnt you just read what he said?

**Comment by Ironbrodiet** [comment]
The link you sent is blocked, but I found another way.

**Reply by SomeRussian72** [reply]
same here link is blocked tbh almost every website my school has been blocked

ngl
